tests/rkt_app_isolator_test.go

```diff
@@ -71,6 +71,10 @@ func TestAppIsolatorMemory(t *testing.T) {
 	rktCmd := fmt.Sprintf("%s --insecure-options=image run --mds-register=false %s", ctx.Cmd(), aciFileName)
 	expectedLine := "Memory Limit: " + strconv.Itoa(maxMemoryUsage)
 	runRktAndCheckOutput(t, rktCmd, expectedLine, false)
+
+	rktCmd = fmt.Sprintf("%s --insecure-options=image run --mds-register=false %s --memory 42Mi", ctx.Cmd(), aciFileName)
```

Comment:
42 Mi = 42 mebibyte = 1048576 bytes.
42 M = 42 megabyte = 1000000 bytes.

The Quantity parser comes from Kubernetes and it understands both.